Formula/beagle.rb

```diff
@@ -3,7 +3,7 @@ class Beagle < Formula
   homepage "https://github.com/beagle-dev/beagle-lib"
   url "https://github.com/beagle-dev/beagle-lib/archive/v3.1.2.tar.gz"
   sha256 "dd872b484a3a9f0bce369465e60ccf4e4c0cd7bd5ce41499415366019f236275"
-  license "LGPL-3.0"
+  license "LGPL-3.0-or-later"
```

Comment:
Some files in the project appear to be licensed under `GPL-3.0-or-later`, such as https://github.com/beagle-dev/beagle-lib/blob/master/m4/ax_gcc_x86_cpuid.m4

Others are licensed under `LGPL-3.0-or-later`, such as https://github.com/beagle-dev/beagle-lib/blob/master/libhmsbeagle/BeagleImpl.h

The entire project seems to fall under LGPL 3.0. I think putting both GPL 3 and later and LGPL 3 and later is appropriate here.

Reply:
If more than one licence identifier is given, are they separated by commas? 

Something like that: `license: "LGPL-3.0-or-later, GPL-3.0-or-later"`

Reply:
Check out the documentation about `all_of:` here: https://docs.brew.sh/License-Guidelines#complex-spdx-license-expressions

`all_of` would likely be more appropriate than `any_of` here. So something like:

```
license all_of: ["LGPL-3.0-or-later", "GPL-3.0-or-later"]
```